Removed Kubernetes API access from `settings update-ca` command

enrichman opened the pull request:
The `epinio` cli should be used without having access to the Kubernetes cluster or the kube API.

The only user command that was left with this access was the `settings update-ca`, looking for the Ingress, Secret and storing the certificate in the settings. This is not needed because the certificate can be loaded directly from the provided address.

This PR is getting the certificate with the same copy/pasted func used by the `login` command, removing some logic and checks (we are always trusting the certificate in this case, and we are checking just the Epinio url, so no need to loop for different addresses).

Tested also locally (messing up the cert in the settings). It was correctly updated.

The `admincmd` and `usercmd` package are now obsolete and could be merged.  
I will leave this for another PR to avoid confusion.

enrichman commented on internal/cli/admincmd/settings.go:
```diff
@@ -115,92 +111,63 @@
-func getEpinioURL(ctx context.Context, cluster *kubernetes.Cluster) (string, string, error) {
-	// Get the ingress
-	ingresses, err := cluster.ListIngress(ctx, helmchart.Namespace(), "app.kubernetes.io/name=epinio")
+	tlsConfig := &tls.Config{InsecureSkipVerify: true} // nolint:gosec // We need to check the validity
```
Dismissing the alert because this command was made for this reason.

andreas-kupries commented:
Looks to have CI issues around the settings command and somewhere around a missing config location (new check).

enrichman commented:
Test should be fixed. The issue was that originally the `settings update-ca` command was `settings update`. When we added the `login` and the OIDC we changed the behaviour of the update to just update the certificate. Some tests were expecting the command to actually still update the whole settings file.

Now the file is manually copied, and the test of the `update-ca` is just checking the generation of the certificate value.
